### DISCUSSION TITLE
[Feature request] bisect command

### DISCUSSION THREAD

**Discussion by chriskrycho** [ember-cli/ember-exam#729] [Ideas]
This is a _large_ feature request, but it would make my life _far_ easier, so I'll throw it out: rspec recently introduced the [`bisect` command](https://relishapp.com/rspec/rspec-core/docs/command-line/bisect), which combines random ordering and seeding to find out which sequence of tests causes a failure in the random-failures situation. Obviously, we can sort of pull this off manually using ember-exam, but that's a time-consuming and error-prone process. Automating it would be _amazing_.

**Comment by trentmwillis** [comment]
Sorry for letting this sit so long. I like this idea and it can likely build off some of the infrastructure provided by the [`exam:iterate` command](https://github.com/trentmwillis/ember-exam/pull/56).

That said, this will likely be very difficult to implement as it involves parsing the output from Testem and transforming it into actionable tasks. If someone is willing to work on it, I'd be happy to provide guidance, but likely won't be getting to it myself any time soon.
